### DISCUSSION TITLE
how do you enable mozc ?

### DISCUSSION THREAD

**Discussion by JunkyardCat** [google/mozc#862] [Q&A]
im using windows 10 and was wondering how to turn the mozc keyboard on?
also how do you change jp input to english input using hotkey?

**Comment by yukawa** [comment]
Have you ever succeeded to use built-in MS-IME Japanese in your environment?

If not, I'd suggest starting from understanding how to enable and use MS-IME Japanese, which should have more articles and documents on the Internet to rely on, e.g. https://www.reddit.com/r/LearnJapanese/.

Once you succeeded to enable and use MS-IME Japanese, it shouldn't be that difficult for you to use Mozc as well, as we've worked hard to make Mozc have similar user experience as MS-IME Japanese.

Hope this helps.
